src/utils/signInHistory.ts

```diff
@@ -0,0 +1,130 @@
+// import { DynamoDB } from "aws-sdk";
+// import { dynamoDBService } from "./dynamo";
+// import { Request } from "express";
+import { activityLogItemsPerPage } from "../config";
+import { prettifyDate } from "./prettifyDate";
+import { ActivityLogEvent } from "./types";
+
+const activityLogEventTypeLocales: Record<string, unknown> = {
+  "signed-in": "signedIn"
+}
+
+// TODO should be in a config somewhere I suppose. 
+// Should be generated using Date.now() whenever a launch date is agreed upon
+const activityLogLaunchDateInMs = 1685032269060;
+
+const data = [
+  {
+    "event_type": "test",
+    "session_id": "asdf",
+    "user_id": "string",
+    "timestamp": "1689210000",
+    "truncated": false
+  }];
+
+export const hasExplanationParagraph = (data: Array<any>) => {
```

Comment:
There seems to be a warning from lint on this line : https://github.com/alphagov/di-account-management-frontend/actions/runs/5209313074/jobs/9398998273?pr=914#step:10:22